mslib/msui/mscolab.py

```diff
@@ -87,7 +87,7 @@ def __init__(self, parent=None, mscolab=None):
 
     def select_archived_operation(self, item):
         logging.debug('select_inactive_operation')
-        if item.access_level == "creator":
+        if item.access_level in ["creator", "admin"]:
```

Comment:
this change must also be done in the UI too, or an admin cannot click the Operation - Maintenance - Archiv in the menu


![need_changes_to_activate_button](https://github.com/user-attachments/assets/b8c4a6e4-9935-4e7f-906e-280891318cd8)

Reply:
There you need to enable the functions on the given role
https://github.com/Open-MSS/MSS/blob/develop/mslib/msui/mscolab.py#L1967

Reply:
nice catch